beacon-chain/cache/committee.go

```diff
@@ -218,3 +156,7 @@ func startEndIndices(c *Committee, index uint64) (uint64, uint64) {
 
 	return start, end
 }
+
+func key(slot uint64, seed [32]byte) string {
```

Comment:
Why is the seed a key? Why the slot when everything else says epoch?

Can you please add commentary on why this is a good key function for committee caching?

Reply:
Using seed and added a comment 👍